exercises/dolica_akelloegwel/ex02_oo_basics/Shapes.cbp

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please don't commit non-cmake project files (i.e. generated files to the project) adding these to a [gitignore](https://git-scm.com/docs/gitignore) file is would be a good idea. Also make sure any generated file go to a separate top-level directory. i.e. use "out of source" builds.